docs/VARIABLES.md

```diff
@@ -2380,6 +2380,86 @@ Default:  "{{mds_ssl_enabled}}"
 
 ***
 
+### sso_mode
+
+SSO mode for C3. Possible values: oidc. if enabling oidc you must set  sso_groups, sso_sub, sso_jwks_uri, sso_authorize_uri, sso_token_uri, sso_issuer_url, sso_client_id, sso_client_password in MDS
+
+Default:  none
+
+***
+
+### sso_groups
+
+Groups in JWT
+
+Default:  none
+
+***
+
+### sso_sub
+
+Sub in JWT
+
+Default:  none
+
+***
+
+### sso_issuer_url
+
+The issuer url, which is typically the authorization server's URL
+
+Default:  none
+
+***
+
+### sso_jwks_uri
+
+JSON Web Key Set (JWKS) URI
+
+Default:  none
+
+***
+
+### sso_authorize_uri
+
+Base uri for authorize endpoint, that initiates an OAuth authorization request
```

Comment:
Can remove base uri part.

Can simply be - Endpoint for an OAuth authorization request